lib/streamlit/elements/media.py

```diff
@@ -74,7 +83,20 @@ def audio(
         """
         audio_proto = AudioProto()
         coordinates = self.dg._get_delta_path_str()
-        marshall_audio(coordinates, audio_proto, data, format, start_time)
+
+        if type_util.is_type(data, "numpy.ndarray") and sample_rate is None:
+            raise StreamlitAPIException(
+                "`sample_rate` must be specified when `data` is a numpy array."
+            )
+        if not type_util.is_type(data, "numpy.ndarray") and sample_rate is not None:
+            import streamlit as st
```

Comment:
@tconkling Explanatory comment added.
In this case, I am not sure that this gives any benefit, I just did this import similar to 
https://github.com/streamlit/streamlit/blob/ddf7e50f13c26ead7d3c7c1c9211c793afeb62a8/lib/streamlit/elements/number_input.py#L235

Please let me know, if this anyway should be imported on the top of the file, I will change this

Reply:
Since streamlit will always already be imported at this point, there's no performance benefit for doing this. The only value IMO is if the local import avoids a circular import error. If moving the import to the top of the file works, then we should do so

Reply:
DONE